### PR TITLE
Add crates.io and docs.rs links to README.md

### PR DESCRIPTION
**Pull request by matthunz**
Congrats everyone on v0.1!

Not sure if this is the best styling to go for, but I think it'd be great to link to the new releases

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 **An experimental Rust architecture for reactive UI**
 
+[![Crate](https://img.shields.io/crates/v/xilem?style=flat-square)](https://crates.io/crates/xilem)
+[![Documentation](https://img.shields.io/badge/docs-latest-blue.svg?style=flat-square)](https://docs.rs/xilem)
```
Comment (DJMcNab) [inline README.md:8]: There are existing badges for these at the bottom of the list of badges, which are commented out.

Could you please change this PR to uncomment those?

**Review by DJMcNab — COMMENTED** [review]
Hi! Thanks for your interest. I agree that this is really important to have!

We had already done some preparation for this change, but hadn't gotten around to uncommenting it. Can you please change this PR to instead use that?

File: README.md (hunk 1 above)
Comment (matthunz) [reply README.md]: Oh wow I didn't even notice😅 sounds good I just switched over

**Review by DJMcNab — APPROVED** [review]
Thanks!

**Pull request merged**